src/app/modules/item/components/permissions-edit-dialog/permissions-edit-dialog.component.ts

```diff
@@ -32,10 +28,20 @@ export class PermissionsEditDialogComponent implements OnInit {
   targetTypeString = '';
 
   ngOnInit(): void {
-    this.targetTypeString = this.targetType.slice(0, -1).toLowerCase();
+    switch (this.targetType) {
+      case 'Users':
+        this.targetTypeString = 'user';
+        break;
+      case 'Groups':
+        this.targetTypeString = 'group';
+        break;
+      case 'Teams':
+        this.targetTypeString = 'team';
+        break;
+    }
```

Comment:
Don't forget to make these to be translatable as well.

Reply:
fixed